src/lib.rs

```diff
@@ -621,6 +621,50 @@ pub mod randr {
     #![allow(clippy::unit_arg)]
     #![allow(clippy::too_many_arguments)]
     include!(concat!(env!("OUT_DIR"), "/randr.rs"));
+
+    /// For information on what the various properties mean, see the [RandR specification][randr-spec]
+    ///
+    /// [randr-spec]: https://cgit.freedesktop.org/xorg/proto/randrproto/tree/randrproto.txt#n1860
+    mod property {
```

Comment:
@RealKC @RealKC This is a _really_ late post-merge review, but I just tried using some of these constants and realize I cannot because the module is private. I'll submit a PR.

Reply:
😅

Reply:
Submitted as https://github.com/rust-x-bindings/rust-xcb/pull/278